python/py-wikipedia-api/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        martin-majlis Wikipedia-API 0.5.4
+name                py-wikipedia-api
+revision            0
+supported_archs     noarch
+license             MIT
+
+python.versions     37 38 39 310
+
+maintainers         nomaintainer
+
+description         ${name} is a python wrapper for Wikipedia's API
+long_description    ${name} is a python wrapper for Wikipedia's API \
+                    that supports extracting text, sections, links, \
+                    categories, translations, etc. for Wikipedia entries.
+
+homepage            https://github.com/martin-majlis/Wikipedia-API
```

Comment:
Nitpick: explicitly specifying `homepage` isn't needed here, since the `github` portgroup automatically sets the homepage off of `githut.setup`.